packages/cloudflare/src/api/memory-queue.spec.ts

```diff
@@ -0,0 +1,52 @@
+import { generateMessageGroupId } from "@opennextjs/aws/core/routing/queue.js";
+import { afterEach, beforeAll, describe, expect, it, vi } from "vitest";
+
+import cache, { DEFAULT_REVALIDATION_TIMEOUT } from "./memory-queue";
+
+vi.mock("./.next/prerender-manifest.json", () => Promise.resolve({ preview: { previewModeId: "id" } }));
+
+const defaultOpts = {
+  MessageBody: { host: "test.local", url: "/test" },
+  MessageGroupId: generateMessageGroupId("/test"),
+  MessageDeduplicationId: "",
+};
+
+describe("MemoryQueue", () => {
+  beforeAll(() => {
+    vi.useFakeTimers();
+    globalThis.internalFetch = vi.fn().mockReturnValue(new Promise((res) => setTimeout(() => res(true), 1)));
+  });
+
+  afterEach(() => vi.clearAllMocks());
+
+  it("should process revalidations for a path", async () => {
+    const firstRequest = cache.send(defaultOpts);
```

Comment:
I would inline defaultOpts in all tests

See DRY vs DAMP, i.e. https://enterprisecraftsmanship.com/posts/dry-damp-unit-tests/